### ISSUE TITLE
DayView selectedDate binding not updating

### ISSUE THREAD

**Issue by barbrick** [Mijick/CalendarView#37]
The selectedDate binding in DayView does not update when changed. This works fine for months but not for days making it impossible to change the selectedDate outside of the touch inputs on the days.

**Comment by purepure** [comment]
yes, actually other params in DayView is not reliable. So i decide to binding these value directly to the DayView:
```
.dayView { date, isCurrentMonthNotReliable, selectedDateThatIWontUse, selectedRange in
            CalendarDayView(date: date, month: month, isCurrentMonth: isCurrentMonthNotReliable, selectedDate: $selectedDate, selectedRange: selectedRange)
        }
```
I'm not sure about the root and also the day seem to be shifted 1 days onward/backward. Anyone know, please make a hints
